### PR TITLE
Add tutorials section

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/tutorials/W&B_Tables_Quickstart.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+# View & Analyze Model Predictions
+
+<a href="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/datasets-predictions/W&B_Tables_Quickstart.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
```
Comment (ngrayluna) [inline docs/tutorials/W&B_Tables_Quickstart.md]: To do: Remove all Google Colab images and replace them w/ text.  e.g. What how I changed it in: Intro_to_Weights_&_Biases.md one.

File: docs/tutorials/Simple_TensorFlow_Integration.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
...KKL7FJZ9du4caNVb/+gyWzn1q1brfaHhdV8Qbtw4UKrvWYdfcz+MOtu9s+gQYPsUln9t3z5cqs/zbb67Nq1S6mpqXI4HLX667PPPtP69evVpUsXderUyS6VVq5cafVl//791bFjR7tUeuedd7Rjxw4lJyfbJVJFRYVVnp2drcTERLtUOnbsmJYuXarCwkLrOj779u2zru/xeNSjRw+7VFbb165dq6ioKHXt2tUuldWHmzZtUq9evRQZGWmXynot0tPTrdcoNDTUKjOvab63zHtceumlVpnJrMP7779v1SkhIcEulVXnDz/80GpD79697VJp586dVn+ZwWVcXJxdKm3YsEGffvqpVRYTE2OXyrrG5s2brWv7h52B7/OmZL7ffO1uy8zX5ItQTx+zvh06tPa6x2gtJ0+etPeaDyOAAQAAAAAAAAAAUAcjgJsHI4AvbJmZmfZe8+jevbsc9j4AAAAAAAAAAAAA4AuOABgAAAAAAAAAAAAA2gkCYAAAAAAAAAAAAABoJwiAAQAAAAAAAAAAAKCdIAAGAAAAAAAAAABAHQ6HQx6PR1VVVXYJzofZj2Z/mv0KNCfeYQAAAAAAAAAAAKgjNDSU8LeJmf1p9ivQnAiAAQAAAAAAAAAAUEdYWJhCQkKqRwETBjeOr+/MfjT70+xXoDkRAAMAAAAAAAAAAKAOc6riwBAY5y4w/GUKaDS3EONNx79WAAAAAAAAAAAABGWGl5WVlTp9+rS1j3NjBr7mtM+EvzBlZmbae3UlJSUpIyPDflS/M53XvXt3AmAAAAAAAAAAAAAAaAlnC4BNZwqBz3aOGQDzZwYAAAAAAAAAAAAA0Eb4Qt5A9ZUHIgAGAAAAAAAAAAAAgDYkMOxtaPhrIgAGAAAAAAAAAAAAgFYWOK2zL/QNDH/Ptk4wATAAAAAAAAAAAAAAtAH1hcA+Zwt/TQTAAAAAAAAAAAAAANBG1BfyNiT8NREAAwAAAAAAAAAAAEAbEhj2NjT8NREAAwAAAAAAAAAAAEAb4wt9zyX8NREAAwAAAAAAAAAAAEAbdK7hr4kAGAAAAAAAAAAAAADaCQJgAAAAAAAAAAAAAGgnCIABAAAAAAAAAAAAoJ0gAAYAAAAAAAAAAACAdoIAGAAAAAAAAAAAAADaCQJgAAAAAAAAAAAAAGgXpP8Pj2oIaI06KFgAAAAASUVORK5CYII=)
```
Comment (ngrayluna) [inline docs/tutorials/Simple_TensorFlow_Integration.md]: Remove this

Comment (scottire) [reply docs/tutorials/Simple_TensorFlow_Integration.md]: Fixed :)

**Comment by cvphelps** [comment]
Adding my edits on a separate PR here https://github.com/wandb/docodile/pull/228

**Pull request merged**